jet_fuel/app/models/url.rb

```diff
@@ -0,0 +1,18 @@
+class Url < ActiveRecord::Base
+  validates :url, presence: true, :format => {:with => URI.regexp}
+  validates :short_url, presence: true, uniqueness: true
+
+  def self.generate
+    randomstring = ((('a'..'z').to_a + (0..9).to_a)*3).shuffle[0,5.to_i].join
```

Comment:
Space missing after comma.<br>Surrounding space missing for operator '*'.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.